docs/operator/deploy.md

```diff
@@ -201,7 +201,7 @@ The flags usage is as follow:
 | `--threshold`      | The minimum number of nodes that need to be online for the network to be live.                                                                                           |
 | `--period`         | The period of the randomness beacon to use. It must be a valid duration as parsed by Golang's [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration) method. |
 | `--catchup-period` | The period of randomness when recovering from a failure. A valid Golang duration, it may be 0 to catch up as fast as possible.                                           |
-| `--secret`         | The password that the leader uses to authenticate nodes that want to participate in the group. This password must be at least 32 characters long.                        |
+| `--secret-file`         | The file that contains the password that the leader uses to authenticate nodes that want to participate in the group. This password must be at least 32 characters long. This variable contains the path and name of the file.                      |
```

Comment:
Would you mind matching the alignment of the md table and deleting 5 of these spaces?
Also we might want to mention that this flag is taking precedence over the env. var if it is set.

Reply:
```suggestion
| `--secret-file`    | The file that contains the password that the leader uses to authenticate nodes that want to participate in the group. This password must be at least 32 characters long. This variable contains the path and name of the file.                      |
```